pkg/watcher/block.go

```diff
@@ -197,9 +198,11 @@ func (w *BlockWatcher) handleBlockInfo(block *BlockInfo) {
 		} else if res.Signed {
 			icon = "✅"
 			w.metrics.ValidatedBlocks.WithLabelValues(block.ChainID, res.Address, res.Label).Inc()
+			w.metrics.ConsecutiveMissedBlocks.WithLabelValues(block.ChainID, res.Address, res.Label).Set(0)
```

Comment:
you should also reset to 0 in the "block proposer" case (the first `if`)

Reply:
Should be fixed by https://github.com/kilnfi/cosmos-validator-watcher/pull/78/commits/96dfbe0ac6681c8bfbc7e992cdce8cde88959ee4